tests/integration/test_example_script.py

```diff
@@ -17,12 +17,12 @@ def mocked_clients(test_mode):
 def test_script():
     initialize()
     # wait until 1 second in future
-    wait((dt.datetime.now() + dt.timedelta(seconds=1)).isoformat())
+    wait_until((dt.datetime.now() + dt.timedelta(seconds=1)).isoformat())
```

Comment:
Is this supposed to actually wait 1 second?  Because that depends on the local timezone...

I propose that utcnow() would be better here.

Reply:
Yup, great point. It would have not waited at all (no tolerance set). I switched to using `utcnow()`. It'll still run very quickly, since we're patching out `time.sleep` to keep the test runs short.